MvvmCross/Core/Binding/Bindings/Target/Construction/MvxTargetBindingFactoryRegistry.cs

```diff
@@ -116,10 +116,16 @@ private IMvxPluginTargetBindingFactory FindSpecificFactory(Type type, string nam
             {
                 return factory;
             }
+            var implementedInterfaces = type.GetTypeInfo().ImplementedInterfaces;
+            foreach (var implementedInterface in implementedInterfaces)
+            {
+                factory = this.FindSpecificFactory(implementedInterface, name);
+                if (factory != null) return factory;
+            }
```

Comment:
I'm not too familiar with this code but shouldn't this be done after the base type?  My gut says that we should check the base class before any interfaces that may be implemented.